client/src/index.css

```diff
@@ -1,9 +1,9 @@
-body {
+/* body {
```

Comment:
Just delete if you don't need it anymore - you can probably delete the `.css` files at some point anyways since you are using styled-components?